NewHorizons/Builder/Props/DialogueBuilder.cs

```diff
@@ -22,36 +22,23 @@ public static (CharacterDialogueTree, RemoteDialogueTrigger) Make(GameObject go,
             var dialogue = MakeConversationZone(go, sector, info, mod.ModHelper);
             
             RemoteDialogueTrigger remoteTrigger = null;
-            if (info.remoteTriggerPosition != null || info.remoteTriggerRadius != 0)
+            if (info.remoteTrigger != null)
             {
-                remoteTrigger = MakeRemoteDialogueTrigger(go, sector, info, dialogue);
+                remoteTrigger = MakeRemoteDialogueTrigger(go, sector, info, info.remoteTrigger, dialogue);
             }
-
-            if (!string.IsNullOrEmpty(info.rename))
+            // Backwards compatibility
```

Comment:
Again, should be in the Migrate method, unless there's some specific reason not to